migrate/v0_15/migrate.go

```diff
@@ -174,14 +175,50 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 			baseStabilityCom := stabilityCom.SetPermissions(newStabilityCommitteePermissions)
 			newStabilityCom := v0_15committee.MemberCommittee{BaseCommittee: baseStabilityCom}
 			committees = append(committees, newStabilityCom)
-		} else {
+		case 2:
 			safetyCom := comtypes.NewMemberCommittee(com.ID, com.Description, com.Members,
 				[]v0_15committee.Permission{v0_15committee.SoftwareUpgradePermission{}},
 				com.VoteThreshold, com.ProposalDuration, v0_15committee.FirstPastThePost)
 			committees = append(committees, safetyCom)
 		}
 	}
 
+	// Initialize hard governance committee
+	hardGovCom := comtypes.TokenCommittee{}
+	hardGovCom.ID = 3
+	hardGovCom.Description = "Hard Governance Committee"
+	hardGovCom.Members = []sdk.AccAddress{sdk.AccAddress("kava1e0agyg6eug9r62fly9sls77ycjgw8ax6xk73es")}
+	hardGovCom.VoteThreshold = sdk.MustNewDecFromStr("0.5")
+	hardGovCom.ProposalDuration = time.Duration(604800000000000)
```

Comment:
nit: explicitly define duration using predefined units (time.Hour, etc) - too easy to make a mistake manually adding zeros

Reply:
Addressed in https://github.com/Kava-Labs/kava/pull/941/commits/25ea9cb6caa3db758836a1f8191ccf586aacae5f